Client/Clique/Common/Models/UserProfile.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  UserProfile.swift
+//  Clique
+//
+//  Created by Infinum on 29.11.2022..
+//
+
+import Foundation
+
+struct UserProfile: Codable {
+    let id: Int?
+    let name: String
+    let surname: String
+    let email: String
+    let contact: String
+    let birth_data: String
+    let profile_pic: String
+    let bio: String
```

Comment:
All fields that can be NULL in the table should be optional here. Also if you used camelCase in the rest of the project why are you using snake_case here for some parameters? 

Also please, please and please again call this model User. It is not user profile, because I'm highly suspicious it is called like that (UserProfile) in the database.